lib/ui/screens/home_screen.dart

```diff
@@ -14,8 +14,10 @@ class HomeScreen extends StatefulWidget {
 
 class _HomeScreenState extends State<HomeScreen> {
   List<Map<String, Object>> _tabs;
+  List<Widget> pages = [CheckInScreen(), ReportScreen(), SettingsScreen()];
   int _selectedTabIndex = 0;
-
+  PageController _pageController;
+  bool isSwipeTab = true;
```

Comment:
now isSwipeTab is always true ??? what is the purpose of isSwipeTab  if it's always true/false ?

Reply:
if not check true/false , we can only be used in swipe mode but not in push
mode below bottomnavigation, I mean always give the user swipe tab, otherwise press tab